governance/votes/Executive Vote - August 2, 2023.md

```diff
@@ -0,0 +1,141 @@
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Enhanced Dai Savings Rate (DSR) Activation
+
+As per this successful governance [poll](https://vote.makerdao.com/polling/QmcTRPLx), the Stability Scope has been amended to include an [Enhanced Dai Savings Rate (EDSR)](https://mips.makerdao.com/mips/details/MIP104#3-2-2-enhanced-dai-savings-rate-edsr-). To achieve this, the DSR will be updated if this executive proposal passes.
+
+- Increase the DSR by 4.81% from 3.19% to **8%**.
+
+Please review the forum [thread](https://forum.makerdao.com/t/request-for-gov12-1-2-edit-to-the-stability-scope-to-quickly-implement-enhanced-dsr/21405) for more information about this change.
+
+### Spark Dai Direct Deposit Module (D3M) Parameter Change
+
+Per this successful governance [poll](https://vote.makerdao.com/polling/QmSLj3HS), the Spark Protocol D3M will be updated if this executive proposal passes.
+
+- Increase the DIRECT-SPARK-DAI [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) by 180 million DAI from 20 million DAI to **200 million DAI**.
+
+The [Target Available Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) and the [Ceiling Increase Cooldown (`ttl`)](https://manual.makerdao.com/module-index/module-dciam#ceiling-increase-cooldown-ttl) will remain unchanged at 20 million DAI and 8 hours respectively.
+
+Please review the forum [thread](https://forum.makerdao.com/t/phoenix-labs-proposed-changes-for-spark/21422) for more information about this change.
+
+### Harbor Trade Credit (RWA004-A) Changes
+
+As per this successful governance [poll](https://vote.makerdao.com/polling/QmR8cYb1), the following changes to the Harbor Trade Credit (RWA004-A) vault will be made if this executive proposal passes.
+
+- Decrease the RWA004-A [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) by 7 million DAI from 7 million DAI to **0 DAI**.
+- `tell` will be called on the RWA Liquidation Oracle for the RWA004-A vault. This will place the vault into a soft liquidation state. This is a technical requirement for Centrifuge to commence the unwinding process.
+
+Please review the forum [thread](https://forum.makerdao.com/t/request-to-poll-decrease-debt-ceiling-for-harbor-trade-credit-htc-drop-to-0/21373) for more information about this change.
+
+### New Silver (RWA002-A) Doc Update
+
+Per this governance [poll](https://vote.makerdao.com/polling/QmaU1eaD), the `doc` parameter for RWA002-A will be updated if this executive proposal passes.
```

Comment:
Use human-readable parameter language. So RWA Agreement versus doc, and link to docs. I agree that the name of that one is bad though.

Reply:
I think you need to give more information here in general. Like, it's not just a doc update, it's agreement to restructure stuff, other parameter changes will come soon, etc.